scopes/dependencies/dependency-resolver/package-manager.ts

```diff
@@ -51,6 +54,13 @@ export interface PackageManager {
     options: PackageManagerResolveRemoteVersionOptions
   ): Promise<ResolvedPackageVersion>;
 
+  getPeerDependencyIssues?(
+    rootDir: string,
+    rootPolicy: WorkspacePolicy,
+    componentDirectoryMap: ComponentMap<string>,
+    installOptions: PackageManagerInstallOptions
```

Comment:
Maybe create a different type for this method?
(It can be for now just equal to the `PackageManagerInstallOptions` type, but better to separate them for outside at least.